Apps/WebClient/src/NewClientApp/src/components/common/HgButtonComponent.vue

```diff
@@ -58,6 +60,7 @@ const classes = computed(() => ({
         :variant="variantDetails.vuetifyVariant"
         :color="variantDetails.color"
         :class="classes"
+        :prepend-icon="prependIcon"
```

Comment:
This is not needed.  Unhandled props are automatically passed to the root element in a template, or, if there are multiple root elements, the element with the `v-bind="$attrs"` prop.